src/chia_log/log_consumer.py

```diff
@@ -164,32 +225,38 @@ def create_log_consumer_from_config(config: dict) -> Optional[LogConsumer]:
     if enabled_consumer == "file_log_consumer":
         if not check_keys(required_keys=["file_path"], config=enabled_consumer_config):
             return None
-        return FileLogConsumer(log_path=Path(enabled_consumer_config["file_path"]))
+
+        if is_win_platform():
+            return WindowsFileLogConsumer(log_path=Path(enabled_consumer_config["file_path"]))
+        else:
+            return PosixFileLogConsumer(log_path=Path(enabled_consumer_config["file_path"]))
 
     if enabled_consumer == "network_log_consumer":
         if not check_keys(
-            required_keys=["remote_file_path", "remote_host", "remote_user"],
-            config=enabled_consumer_config,
+            required_keys=["remote_file_path", "remote_host", "remote_user"], config=enabled_consumer_config
         ):
             return None
 
-        # default SSH Port : 22
-        remote_port = enabled_consumer_config.get("remote_port", 22)
```

Comment:
Intentionally dropping support for the custom port 🤔

Reply:
Wrong cherry pick... reverted